tests/foreman/ui/test_errata.py

```diff
@@ -757,3 +760,74 @@ def test_positive_filtered_errata_status_installable_param(session, errata_statu
                 if key in expected_values
             }
             assert actual_values == expected_values
+
+
+@tier3
+def test_content_host_errata_search_commands(session, module_org, module_repos_col):
+    """ View a list of affected content hosts for security (RHSA) and bugfix (RHBA) errata,
+    filtered with errata status and applicable flags. Applicability is calculated using the
+    Library, but Installability is calculated using the attached CV, and is subject to the
+    CV's own filtering.
+
+:id: 45114f8e-0fc8-4c7c-85e0-f9b613530dac
```

Comment:
Haven't noticed that before, but metadata (i.e. everything down to end of docstring, including closing `"""`) should be indented by one level.

Reply:
Yes, fixed now, thank you